README.md

```diff
@@ -216,6 +217,27 @@ due to the inappropriate recursive nature of `assert.Equal` (based on
 
 ---
 
+### contains
+
+```go
+❌
+assert.True(t, strings.Contains(a, "abc123"))
+assert.False(t, !strings.Contains(a, "abc123"))
```

Comment:
- supported negation
- removed `string(b)` case, because this is the same as just string (no transformations here)
- separated invalid Contains/NotContains cases with newline (see `README#empty` as example)